--- origsrc/luasoap-1.0b/http.lua	2004-12-02 11:39:36.000000000 -0600
+++ src/luasoap-1.0b/http.lua	2008-02-13 20:36:34.593750000 -0600
@@ -4,7 +4,7 @@
 -- $Id: lua-soap-1.0b-1.src.patch,v 1.1 2008-02-14 02:39:11 yselkowitz Exp $
 ---------------------------------------------------------------------
 
-require"luasocket"
+require"socket"
 require"soap"
 
 local post = socket.http.post
--- origsrc/luasoap-1.0b/soap.lua	2004-12-02 11:39:36.000000000 -0600
+++ src/luasoap-1.0b/soap.lua	2008-02-13 20:34:07.203125000 -0600
@@ -12,7 +12,7 @@
 local max = math.max
 local parse = lxp.lom.parse
 
-module (arg and arg[1])
+module ("soap")
 
 _COPYRIGHT = "Copyright (C) 2004 Kepler Project"
 _DESCRIPTION = "LuaSOAP provides a very simple API that convert Lua tables to and from XML documents"
--- origsrc/luasoap-1.0b/test.lua	2004-12-02 11:39:36.000000000 -0600
+++ src/luasoap-1.0b/test.lua	2008-02-13 20:35:08.187500000 -0600
@@ -4,7 +4,7 @@
 -- $Id: lua-soap-1.0b-1.src.patch,v 1.1 2008-02-14 02:39:11 yselkowitz Exp $
 ---------------------------------------------------------------------
 
-require"lxp/lom"
+require"lxp.lom"
 require"soap"
 
 function table.equal (t1, t2)
@@ -190,8 +190,8 @@
 for i, t in ipairs(tests) do
 	local s = soap.encode (t.namespace, t.method, t.entries, t.header)
 	s = string.gsub (s, "[\n\r\t]", "")
-	local ds = assert (lom.parse ([[<?xml version="1.0" encoding="ISO-8859-1"?>]]..s))
+	local ds = assert (lxp.lom.parse ([[<?xml version="1.0" encoding="ISO-8859-1"?>]]..s))
 	t.xml = string.gsub (t.xml, "[\n\r\t]", "")
-	local dx = assert (lom.parse ([[<?xml version="1.0" encoding="ISO-8859-1"?>]]..t.xml))
+	local dx = assert (lxp.lom.parse ([[<?xml version="1.0" encoding="ISO-8859-1"?>]]..t.xml))
 	print (table.equal (ds, dx))
 end
